crates/context_aware_config/src/api/default_config/handlers.rs

```diff
@@ -184,3 +194,59 @@ async fn get(db_conn: DbConnection) -> superposition::Result<Json<Vec<DefaultCon
     let result: Vec<DefaultConfig> = default_configs.get_results(&mut conn)?;
     Ok(Json(result))
 }
+
+pub fn get_key_usage_context_ids(
+    key: &str,
+    conn: &mut PooledConnection<ConnectionManager<PgConnection>>,
+) -> superposition::Result<Vec<String>> {
+    let result: Vec<Context> = contexts.load(conn).map_err(|err| {
+        log::error!("failed to fetch contexts with error: {}", err);
+        db_error!(err)
+    })?;
+
+    let mut context_ids = vec![];
+    for context in result.iter() {
+        from_value::<Map<String, Value>>(context.override_.to_owned())
+            .map_err(|err| {
+                log::error!("failed decode override into object: {}", err);
+                unexpected_error!(err)
```

Comment:
Passing any parsing error to the user is not needed, that's the point I am trying to make.

Reply:
yes, but unexpected_error anyway ignores messages in the implementation

Reply:
Ok sure